aiida/utils/fixtures.py

```diff
@@ -3,9 +3,14 @@
 
 Usage (unittest)::
 
+    import os
+
     MyTestCase(aiida.utils.fixtures.PluginTestCase):
 
-        BACKEND = <load from env variable etc>
+        BACKEND = os.environ.get('TEST_BACKEND')
+        # load the backend to be tested from the environment variable
+        # proceed the test command with TEST_BACKEND='django' | 'sqlalchemy'
```

Comment:
simply prepend the test command by TEST_BACKEND='django'  or TEST_BACKEND='sqlalchemy'